APIs/openEO/fair.qmd

```diff
@@ -9,23 +9,26 @@ aliases:
 
 ## FAIR
 
-One of the key goals of openEO, is to support [FAIR principles](https://www.go-fair.org/fair-principles/) and open science. The implementation in the Copernicus dataspace
-makes it easier to comply with these principles, by incorporating these principles in the implementation, so that
-users are automatically a step closer to generating FAIR-compliant open data. If your project has requirements related to these topics,
+One of the key goals of openEO, is to support [FAIR principles](https://www.go-fair.org/fair-principles/) and open science. Directly implementing these principles in Copernicus Dataspace makes it easier to follow them. In other words,  with the use of openEO, users are creating FAIR-compliant open solutions automatically. If your project has requirements related to these topics,
```

Comment:
the meaning gets changed here from:
being a step closer to fair-compliant data
to:
automatically generating fair compliant data

This is not intended as pure marketing, so I prefer to keep the nuance.